tools/cmake/modules/FindSDLLibUSB.cmake

```diff
@@ -9,11 +9,19 @@ if(${OS} STREQUAL "posix")
   set(LIBUSB_LIBRARIES usb-1.0 udev)
 elseif(${OS} STREQUAL "win")
   get_arch(ARCH)
-  if(${ARCH} STREQUAL "x86")
-    set(LIBUSB_DIR ${CMAKE_SOURCE_DIR}/build/libusbx_win_x86)
-  elseif(${ARCH} STREQUAL "x64")
-    set(LIBUSB_DIR ${CMAKE_SOURCE_DIR}/build/libusbx_win_x64)
-  endif()
+    if(NOT EXISTS $ENV{SDL_LIBUSB_DIR})
+      if(${ARCH} STREQUAL "x86")
+        set(LIBUSB_DIR ${CMAKE_SOURCE_DIR}/build/libusbx_win_x86)
+      elseif(${ARCH} STREQUAL "x64")
+        set(LIBUSB_DIR ${CMAKE_SOURCE_DIR}/build/libusbx_win_x64)
+      endif()
+    elseif(EXISTS $ENV{SDL_LIBUSB_DIR})
+      if(${ARCH} STREQUAL "x86")
+        set(LIBUSB_DIR $ENV{SDL_LIBUSB_DIR}/libusbx_win_x86)
+      elseif(${ARCH} STREQUAL "x64")
+        set(LIBUSB_DIR $ENV{SDL_LIBUSB_DIR}/libusbx_win_x64)
+      endif()
+    endif() 
```

Comment:
@antongovorov if you'll check environment variable in cmd file and set it properly there you will not need this check at all. You can just read environment variable here